lib/identity/config.js

```diff
@@ -26,11 +26,7 @@
 import { DEFAULT_API_URL } from '../utils/constants'
 
 /**
- * Configuration and credentials for E3DB.
- *
- * @property {string} realmName The realm's globally unique name
```

Comment:
Did you mean to delete these?

Reply:
yea -- parameters are already defined by the constructor method -- no need to define them in the class docblock as well.